draft-ietf-taps-interface.md

```diff
@@ -2344,7 +2353,9 @@ based on the Maximum Packet Size (MPS). The value of this property can change ov
 This value allows a sending stack to avoid unwanted fragmentation at the
 network-layer or segmentation by the transport layer before
 choosing the message size and/or after a `SendError` occurs indicating
-an attempt to send a Message that is too large.
+an attempt to send a Message that is too large.  A Transport Services API
+could express `Not applicable` in various ways -- e.g., by letting a querying
+function call fail.
```

Comment:
```suggestion
could express `Not applicable` in an environment-typical way, e.g., as a Union type or special value.
```

Reply:
Wait wait, this one is different!  It's a read-only parameter, where "not applicable" is returned... do you want to say "Union type or special value" here?  Ok with me but just checking... I thought a failing query function is easy enough?

Reply:
you are right, it's different. But the proposed variant "letting a querying function call fail." is also no really good solution. I also prefer as a Union type or special value here.